src/components/input/Input.tsx

```diff
@@ -1,10 +1,21 @@
 import { ComponentProps } from "react";
-import styles from "./index.module.css";
+import styles from "./Index.module.css";
+import classNames from "classnames";
 
 interface IInputProps extends Omit<ComponentProps<"input">, "type"> {
   type?: "text" | "number" | "email" | "password" | "url";
+  variant?: "normal" | "error" | "";
 }
 
-export function Input({ type, ...props }: IInputProps) {
-  return <input type={type} className={styles.input} {...props} />;
+export function Input({ type, variant = "normal", ...props }: IInputProps) {
+  return (
+    <input
+      type={type}
+      className={classNames(styles.input, {
+        [styles.input]: variant === "normal",
+        [styles.errorInput]: variant === "error",
```

Comment:
When implementing the `error` prop. You could do this as followed:

`[styles.error]: error`